packages/graphile-build-pg/src/QueryBuilder.js

```diff
@@ -465,11 +465,17 @@ class QueryBuilder {
         this.compiledData.orderBy.length
           ? sql.fragment`order by ${sql.join(
               this.compiledData.orderBy.map(
-                ([expr, ascending]) =>
+                ([expr, ascending, nullsFirst]) =>
                   sql.fragment`${expr} ${
                     Number(ascending) ^ Number(flip)
                       ? sql.fragment`ASC`
                       : sql.fragment`DESC`
+                  }${
+                    nullsFirst === true
+                      ? sql.fragment` NULLS FIRST`
+                      : nullsFirst === false
+                        ? sql.fragment` NULLS LAST`
+                        : null
```

Comment:
I had to actually read the source to make sure `null` was okay here! [It is ✅](https://github.com/graphile/pg-sql2/blob/d99ed2313976b1193eed2ff03f6639c8a04ac32b/src/index.ts#L193)

Reply:
Yeah, I find that really handy..  don't change it! :smile: